src/client/datascience/notebook/vscodeNotebookController.ts

```diff
@@ -325,7 +325,14 @@ export class VSCodeNotebookController implements Disposable {
             join(this.context.extensionPath, 'out', 'datascience-ui', 'ipywidgetsKernel', 'require.js'),
             join(this.context.extensionPath, 'out', 'ipywidgets', 'dist', 'ipywidgets.js'),
             join(this.context.extensionPath, 'out', 'datascience-ui', 'ipywidgetsKernel', 'ipywidgetsKernel.js'),
-            join(this.context.extensionPath, 'out', 'fontAwesome', 'fontAwesomeLoader.js')
+            join(this.context.extensionPath, 'out', 'fontAwesome', 'fontAwesomeLoader.js'),
+            join(
```

Comment:
What's this script addition here?

Reply:
Oh crap. Forgot about this. I was trying to debug something before. Thanks. I'll remove it.

Reply:
Yeah, looked like a debug something. Rest looks good.